README.md

```diff
@@ -199,7 +201,9 @@ If you want to run it on MAC, you should follow [this Instruction](MacGPUEnv.md)
 
 # Related Resources
 ### Replicate demo of PhotoMaker: 
-[Demo link](https://replicate.com/jd7h/photomaker) by [@yorickvP](https://github.com/yorickvP), transfer PhotoMaker to replicate.

```

Comment:
This is a very suspicious, Why are you deleting the whole thing? 
### How can I get myself unsuspicious?
Please leave a comment and tell me why did you delete the whole thing That is really suspicious.